mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule OpenAPI.MixProject do
     [
       app: :oapi_generator,
       version: @version,
-      elixir: "~> 1.13",
+      elixir: "~> 1.14",
```

Comment:
With the assumption that you want to support the latest three stable versions, as I observed in the `.github/workflows/ci.yml` file.